### DISCUSSION TITLE
Welcome to ThRasE Discussions!

### DISCUSSION THREAD

**Discussion by XavierCLL** [SMByC/ThRasE#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ms22243** [comment]
Is there a way to import a shapefile of points with edit comments and coordinates to use as a reference for edits? I have a table of 400 edits to make and am able to overlay it on top of my thematic raster within the ThRasE tool but I can not find an easy way to jump from point to point while seeing each edit criteria?

**Reply by ms22243** [reply]
I was able to figure out how to create a bounding tile around the points to jump from point to point, but was wondering if there is a way to show the attribute table of those points so I can see comments as I go through

**Reply by XavierCLL** [reply]
Hi @ms22243 

Yes, that is possible, there are multiples ways to do that, here is one of them:

Go to edit the shapefile of points and change the marker to "Font Marker":

> ![image](https://user-images.githubusercontent.com/11801453/231910092-59c4df94-d535-4073-9635-60e4b62a3ac4.png)

Set the character to display as text using a field in the table for that shapefile in this option:

> ![image](https://user-images.githubusercontent.com/11801453/231910316-4bef59d3-3658-4fd9-9ce9-2959f41b51fd.png)

In Thrase, go to navigation by tiles, select by points and set the shapefile, in the layers in the main window you can select the shapefile of points over the raster to edit, and maybe if you want you can duplicate the shapefile if you want a buffer around the point as a different layer:

> ![image](https://user-images.githubusercontent.com/11801453/231910748-18f15a86-e7a4-4fea-b98f-db3dca59295d.png)

I hope that works for you, regards,
Xavier

(next time please open a new discussion, for specific questions)

**Reply by ms22243** [reply]
Thanks for the detailed reply! Very helpful.
